### PR TITLE
5.2.0 validations

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

Add additional validations to reflect new chain rules introduced in 5.2.0

### Breaking Changes

Although not breaking API changes, these deserve a mention:
  - Portfolio movements can only move the same Asset once
  - New tickers must consist of alpha numeric values only now

### JIRA Link

[DA-494](https://polymesh.atlassian.net/browse/DA-494)

For the point:

- Add ensure check for revoke_claim to make sure claim exist

I tried and we will return the error `PolymeshError: Attempt to edit claims that weren't issued by the signing Identity`

This is an accurate message, although in theory, it could be more precise for this specific case.

### Checklist

- [ ] Updated the Readme.md (if required) ?

[DA-494]: https://polymesh.atlassian.net/browse/DA-494?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/utils/internal.ts [hunk 1]
```diff
@@ -1391,6 +1401,22 @@ export function assertTickerValid(ticker: string): void {
   }
 }
 
+/**
+ * @hidden
+ *
+ * Validates a ticker value is valid to be created. Some tickers were created with more relaxed rules
+ */
+export function assertTickerCreatable(ticker: string): void {
```
Comment (VictorVicente) [inline src/utils/internal.ts:1409]: Not a fan of this name. `assertTickerIsValid`?

Comment (polymath-eric) [reply src/utils/internal.ts:1409]: `assertTickerValid ` is already defined, and we need to leave that function as is to support grandfathered tickers.

Comment (polymath-eric) [reply src/utils/internal.ts:1409]: > The new restriction is only on new ticker registrations/create_asset calls.  All other usages of Ticker still have the old validation.

I think `create` captures the intended use case of where this validation should be used.

Comment (polymath-eric) [reply src/utils/internal.ts:1409]: Looking at it, this check can be inlined and I can get rid of this function

File: src/api/procedures/moveFunds.ts [hunk 2]
```diff
@@ -82,6 +83,17 @@ export async function prepareMoveFunds(
     });
   }
 
+  const tickers = items.map(({ asset }) => (typeof asset === 'string' ? asset : asset.ticker));
```
Comment (prashantasdeveloper) [inline src/api/procedures/moveFunds.ts:86]:
```suggestion
  const tickers = items.map(asTicker);
```

**Review by prashantasdeveloper — CHANGES_REQUESTED** [review]
Left a small nit comment

**Review by prashantasdeveloper — APPROVED** [review]

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**